api/src/pcapi/core/offers/api.py

```diff
@@ -958,10 +977,15 @@ def update_offer_fraud_information(offer: AnyOffer, user: users_models.User | No
     offer.lastValidationType = OfferValidationType.AUTO
     offer.lastValidationAuthorUserId = None
 
-    if offer.validation in (models.OfferValidationStatus.PENDING, models.OfferValidationStatus.REJECTED):
-        offer.isActive = False
+    is_either_pending_or_rejected = offer.validation in (
+        models.OfferValidationStatus.PENDING,
+        models.OfferValidationStatus.REJECTED,
+    )
+    if is_either_pending_or_rejected:
+        _deactivate_any_offer(offer)
     else:
-        offer.isActive = True
+        _activate_any_offer(offer)
+
```

Comment:
High logaf : suspicion de complexité inutile

Je pense que tu n'as pas besoin de ces 2 helper fonctions ici. Notamment, on ne leur passe que des `Offer` donc pas besoin de gérer le cas où `isinstance(offer, models.Offer)` est faux

Proposition: 
```suggestion
    # Cas PENDING : on veut garder la valeur de publicationDatetime pour quand l'offre sera validée.
    # Cas REJECTED : on peut supprimer publicationDatetime, l'offre ne sera jamais publiée
    # Tous les autres cas : ne rien faire. L'activation est déjà portée par la valeur de publicationDatetime
    # On se retrouve avec une seule condition simple (note: les commentaires ci-dessus ne sont pas une proposition de lignes à ajouter. Juste du détail de discussion)
    if offer.validation == models.OfferValidationStatus.REJECTED:
        offer.publicationDatetime = None